### PR TITLE
test on commit

### PR DESCRIPTION
**Pull request by NestorHernandezA**
comment number 123

### CODE REVIEW

**Comment by acwiesn** [comment]
testing testing

File: README.md [hunk 1]
```diff
@@ -1,4 +1,3 @@
 # testPOC
 [![Coverage Status](https://coveralls.io/repos/github/NestorHernandezA/testPOC/badge.svg?branch=master)](https://coveralls.io/github/NestorHernandezA/testPOC?branch=master)
-Cange for test purposes, testing pull requests and git status
```
Comment (NestorHernandezA) [inline README.md]: What a test

**Comment by NestorHernandezA** [comment]
testaagain

**Comment by NestorHernandezA** [comment]
Me too

**Comment by NestorHernandezA** [comment]
TEST CAME FROM API!

**Comment by NestorHernandezA** [comment]
NEW TEST

**Comment by NestorHernandezA** [comment]
commentusingcurl

**Comment by NestorHernandezA** [comment]
(+) No known vulnerabilities found > LUSA-Jenkins-Testing@0.0.0 test /scratch/jenkins/workspace/Lilly USA Digital/stage-lusa jenkins project  > mocha -R spec ./test/test.js Your site is running on port 5000 LUCID Web Pattern ✓ should use lucid web config and lucid web redirects ✓ should handle bad routes ✓ should set a templating engine, currently handlebars only 3 passing (49ms)

**Comment by NestorHernandezA** [comment]
(+) No known vulnerabilities found > LUSA-Jenkins-Testing@0.0.0 test /scratch/jenkins/workspace/Lilly USA Digital/stage-lusa jenkins project > mocha -R spec ./test/test.js Your site is running on port 5000 LUCID Web Pattern ✓ should use lucid web config and lucid web redirects ✓ should handle bad routes ✓ should set a templating engine, currently handlebars only 3 passing (49ms)

**Comment by NestorHernandezA** [comment]
(+) No known vulnerabilities found > LUSA-Jenkins-Testing@0.0.0 test /scratch/jenkins/workspace/Lilly USA Digital/stage-lusa jenkins project  > mocha -R spec ./test/test.js Your site is running on port 5000 LUCID Web Pattern ? should use lucid web config and lucid web redirects ? should handle bad routes ? should set a templating engine, currently handlebars only 3 passing (49ms)

**Comment by NestorHernandezA** [comment]
TEST CAME FROM API!

**Pull request merged**